doc/index.rst

```diff
@@ -45,8 +45,10 @@ for writing Haskell code that can be compiled to Plutus Core.
 .. toctree::
    :caption: Architectural decision records
    :maxdepth: 1
+   :titlesonly:
 
-   adr/index
+   0001-record-architecture-decisions
+   0002-steppable-cek-machine
```

Comment:
Did you mean to do this? I think this makes the other index file redundant since you just flattened the hierarchy?

Reply:
I was just following the instruction on the ADR index.rst. Should I have?

Reply:
if we don't want to I'll update index.rst too.

Reply:
Which instruction? It says to add it to the TOC, which you did by adding it to `adr/index.rst`. Here you've replaced the TOC here that referenced that `index.rst` with something that lists all the ADRs directly, and I'm not sure why...

Reply:
Oops, actually I think I got the files mixed up. Thanks.